### PR TITLE
Mobile base styling

### PR DESCRIPTION
**Pull request by fakesamgregory**
Merge #5 first.

### CODE REVIEW

File: src/assets/sass/_global.scss [hunk 1]
```diff
@@ -16,8 +18,19 @@ h2 {
   border-bottom: 4px solid;
   font-size: 32px;
   padding-bottom: 10px;
-  width: 350px;
   margin-bottom: 5px;
+  text-align: center;
+
```
Comment (suzannahmahala) [inline src/assets/sass/_global.scss]: should this space be removed?

File: src/components/Events.vue [hunk 2]
```diff
@@ -57,5 +68,6 @@
     text-decoration: underline;
     font-family: $font-raleway;
     font-size: 18px !important;
```
Comment (suzannahmahala) [inline src/components/Events.vue]: How come this has been set to 18px? Also we have variables for font-sizes.

File: src/assets/sass/_global.scss (hunk 1 above)
Comment (fakesamgregory) [reply src/assets/sass/_global.scss]: @suzannahmahala what space? Are you referring to the entirety of line 23? I guess this is personal preference (not mine)

File: src/components/Events.vue (hunk 2 above)
Comment (fakesamgregory) [reply src/components/Events.vue]: @suzannahmahala I'm not sure what the font size is. I can add it if you tell me what it is, or we can address when coming to refine the Events component. Non of the components are final in this branch. I've just been addressing the base styling for mobile.

**Pull request merged**